test/facets/third-party-facet.spec.ts

```diff
@@ -242,14 +243,14 @@ describe("Contract: ThirdPartyFacet", () => {
 
         const tx = thirdPartyFacet.connect(thirdParty).accuse(
           sarcoId,
-          archaeologists.slice(0, threshold).map(a => a.hashedShard),
+          archaeologists.slice(0, threshold).map(a => hashBytes(a.unencryptedShard)),
```

Comment:
Sorry if this is a dumb question but is this meant to be an unencrypted shard *hash*, or the plaintext shard?

Reply:
@DarksightKellar -- `accuse` takes the shard single hash as proof.

This code that was updated here is functionally the same as before. I had to make the update b/c the `createSarcoFixture` no longer returns the shard single hash -- instead it returns the shard double hash (since that is used during `create` now, and we only need the single hash during accuse).

Reply:
Ahhh, gotcha